### ISSUE TITLE
The structure editor is not as good as PQS

### ISSUE THREAD

**Issue by pjknowles** [molpro/iMolpro#15]

**Comment by klauspdoll** [comment]
Yes, I have deleted everything, and now I can't figure out how to build a molecule. I can choose "O" , but don't see how to add this to the empty structure. 

And "Undo" or "Ctrl-Z" does not work for me. 

And when the structure is empty and I save it, then it is not an empty file or so, but the previous structure.
